### ISSUE TITLE
Update `guides/context_expressions.md` with new values after #4499 merge

### ISSUE THREAD

**Issue by Carreau** [napari/napari#4590]
From #4499, 

> I do no see update of [guides/context_expressions.md](https://github.com/napari/napari/blob/e39b79bea1671b9a265e9c55316415da0ede1e19/docs/guides/contexts_expressions.md#napari-context-keys) in this PR, but this file contains table of example context keys that will be outdated after this PR merge. Could you update it? 

> Is there place where all context keys are documented? or only code and PR comments (I'm not familiar with this part of napari code)?

_Originally posted by @Czaki in https://github.com/napari/napari/pull/4499#pullrequestreview-976164170_

@talmbert03 later replied:

> That doc can be considered internal for napari core devs (as stated at the top). I do have a plan for (auto)generating front facing docs using the descriptions jn these ContextKey objects... But until a plugin can actually take advantage of these keys using when conditions, we're not going to break anything by changing these here

As we may have this document auto generated and the plugins do not yet have access to those values, I think we can postpone the documentation update and move forward with the code.

**Comment by Carreau** [comment]
Thanks, I had the impression that these comments in the #4499 review were the blocking one, so I opened. this just to be safe. It's easy to close later if we want. Especially since GH now have "close as not planned".
